docs/authorization.md

```diff
@@ -0,0 +1,166 @@
+After evaluating each policy, the authorizer combines the results to make an authorization decision. It does so by applying the following rules:
+
+1. If any `forbid` policy evaluates to `true`, then the final result is `Deny`.
+
+2. Else, if any `permit` policy evaluates to `true`, then the final result is `Allow`.
+
+3. Otherwise (i.e., no policy is satisfied), the final result is `Deny`.
+
+The authorizer returns an _authorization response_, which includes its decision along with some diagnostics. These diagnostics include the _determining policies_ and any _error conditions_. In the case the decision is `Deny`, the determining policies include any `forbid` policies that satisfy the request (rule 1). In the case the decision is `Allow`, the determining policies are the `permit` policies that satisfy the request (rule 2). If no policy is satisfied, the determining policies will be empty. Whatever the final result, if the evaluation of any policies resulted in `error`, then the IDs of the erroneous policies are included in the diagnostics, too, along with the particulars of the errors.
+
+### Discussion<a name="request-authorization-discussion"></a>
+
+Cedar's authorization algorithm has three useful properties:
+
+1. **default deny**: no request with be authorized (decision `Allow`) unless there is a specific `permit` policy that grants it; by default, the decision will be `Deny`. 
+1. **forbid overrides permit**: even if a `permit` policy is satisfied, any satisfied `forbid` policy will _override_ it, producing a `Deny` decision. 
+1. **skip on error**: if a policy's evaluation returns `error`, the policy does not factor into the authorization response; it is skipped.
+
+Why was Cedar's authorization algorithm designed to satisfy these properties? The first two properties make Cedar policies easier to understand. Since `permit` policies are the only way access is granted, readers just have to understand what each policy says, not what it does not. Because `forbid` policies _always_ deny access, readers can understand them independently of any `permit` policies created now or in the future; `forbid` policies effectively define permission "guardrails" that `permit` policies cannot cross. 
+
+The reasoning for the skip-on-error property is more involved, and is discussed in this [blog post](https://cedarland.blog/design/why-ignore-errors/content.html) written by one of the Cedar designers. An alternative authorization algorithm we considered would be to `Deny` a request when any policy evaluation exhibits an error. While this may sound good at first, deny-on-error raises concerns of _safety_: An application that was working fine with 100 policies might suddenly start denying _all_ requests if the 101st policy has an error in it. Skip-on-error avoids this dramatic failure mode, and more flexible: Applications can always choose to look at the authorization response's diagnostics and take a different decision if an evaluated policy produces errors.
```

Comment:
Drop "written by one of the Cedar designers."

I think this is an "availability" rather than "safety" issue. 

Typo: "and [is] more flexible"